packages/legend-application-query/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -373,6 +374,7 @@ export const QueryBuilderResultPanel = observer(
     const resultState = queryBuilderState.resultState;
     const queryParametersState = queryBuilderState.queryParametersState;
     const executionResult = resultState.executionResult;
+    const [isRunningQuery, setIsRunningQuery] = useState(false);
```

Comment:
we don't need the flag `isRunningQuery` at all. Please help me rename `QueryBuilderResultState.isExecutingQuery` to `isRunningQuery` and use that flag instead.